check_dirty.py

```diff
@@ -0,0 +1,18 @@
+"""
```

Comment:
I consider it best practice to include a "shebang" here: https://realpython.com/python-shebang/#whats-a-shebang-and-when-should-you-use-it

Reply:
It has to be the first row of the file I think.